packages/webapp/components/layouts/AccountLayout/EmailSentSection.tsx

```diff
@@ -0,0 +1,46 @@
+import { Button } from '@dailydotdev/shared/src/components/buttons/Button';
+import { Overlay } from '@dailydotdev/shared/src/components/utilities';
+import React, { ReactElement } from 'react';
+import { OverlayContainer } from './common';
+
+interface EmailSentSectionProps {
+  onResend?: (e: React.MouseEvent) => void;
+  onCancel?: (e: React.MouseEvent) => void;
+}
+
+function EmailSentSection({
+  onResend,
+  onCancel,
+}: EmailSentSectionProps): ReactElement {
+  return (
+    <OverlayContainer className="mt-6 border-theme-status-warning">
```

Comment:
Why don't we leverage modals for this?

Reply:
Ah, this is not a modal, this is the faded background overlay in the image below. Probably a poor naming, but I can't think of anything other than "Overlay".
![image](https://user-images.githubusercontent.com/13744167/183612977-4f480d4e-261a-4d68-abe1-11280e0242be.png)

Reply:
Oh right, yeah overlay seems a tad off then.
Perhaps we can leverage `AlertContainer` we have something similar coming up for the community picks.
Although that has a border side to it.
(See here: https://dailydotdev.slack.com/archives/C03JJ8CBEH3/p1659967066043279)

Do you think we can make one generic "AlertContainer" that can handle all of these?

Reply:
I think, yes. Alert container sounds about right since we are trying to get the user's attention with this. An alert fits the context. Will refactor this.